PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+Hello,
+
+Thank you for submitting a pull request, that's awesome! Please add some info (if applicable) to give us some context on the PR.
+
+Things to consider before you submit the PR:
+
+* Has `CHANGELOG.md` been updated? (optional)
```

Comment:
I agree with @jwallra that probably not a lot of people will do it anyways. Let's remove this line, so that we keep only the things that really matter.